### ISSUE TITLE
Edited: BUG: comments in \<focus> lines will not be stripped

### ISSUE THREAD

**Issue by s0cr4tes** [wasiejen/Free-Snap-Tap#60]
Hey man, great work.

But when I launch the game it doesn't work.   Red dot top corner, so the update is the right one. 
Tried both with admin rights and double right click.

>> NO FOCUS APP FOUND when I ALT TAB - Looking for Counter, etc.. 

Old versions work fine.

I did a fresh download of 1.0.4, gonna use snap tap for CS2 - no other games -  and set up my FST-cfg like this:

<arg>-status_indicator=20
<arg>-aliasdelay=50,50
<arg>-tapdelay=10,5
<arg>-nomenu
<arg>-nocontrols
<focus>Counter

<focus>visual
<arg>-crosshair=-10,-10
# Tap Groups
a, d
w, s
# Rebinds
#c : shift
shift : h
# Macros
v::b,u,r,b
#+vert_scroll   :: t|0
#-vert_scroll   :: g|0
#+hori_scroll   :: f|0
#-hori_scroll   :: h|0

-vert_scroll :: x|0
+vert_scroll :: y|0

-mouse_x2 :: g
+mouse_x2 :: h

<focus>sublime
<arg>-crosshair=60,0

v::h,i,h,o

What am i missing?=

**Comment by wasiejen** [comment]
heyho, the last macros replace your press of w, s, a and d with the release of these keys - so no presses will be registered because rebinds and macros suppress their trigger key before it can get to the tap group handling.

> w:: +w s:: +s a:: +a d:: +d

if you want to adjust the presses and do extra things you always have to:

```bash
w :: -w, ... # adding stuff to do additionally here
# w as trigger in this case is the same the press of w = -w```

**Comment by s0cr4tes** [comment]
Hey man, thanks.

Can you help me with the config?
I cant get it to work.

<arg>-status_indicator=20
<arg>-aliasdelay=50,50
<arg>-tapdelay=10,5
<arg>-nomenu
<arg>-nocontrols

<focus>Counter 

# Tap Groups
w, s
a, d

# Basic Rebinds
shift : h

# Simple Testing Macros
+w|(tr("+w")>100)|(last("-space")>500), !s, !ctrl ::  +w|15|5, -s|(cs("+w")*1.1), +s|0|0
+s|(tr("+s")>100)|(last("-space")>500), !w, !ctrl ::  +s|15|5, -w|(cs("+s")*1.1), +w|0|0
+a|(tr("+a")>100)|(last("-space")>500), !d, !ctrl ::  +a|15|5, -d|(cs("+a")*1.1), +d|0|0
+d|(tr("+d")>100)|(last("-space")>500), !a, !ctrl ::  +d|15|5, -a|(cs("+d")*1.1), +a|0|0

# trigger group function:
# |(tr("+w")>100          # only trigger if w was pressed for at least 100 ms
# |(last("-space")>500)   # do not trigger the macro if space was pressed in the last 500 ms - so still jumping
# , !s, !ctrl                       # do not trigger if s and/or control are pressed

**Comment by s0cr4tes** [comment]
I still cant get it to work. NOw i added this in my FST.
What is it that im missing?

<arg>-status_indicator=20
<arg>-aliasdelay=50,50
<arg>-tapdelay=10,5
<arg>-nomenu
<arg>-nocontrols

<focus>Counter  # Adjust this as needed

# Tap Groups
w, s
a, d

# Basic Rebinds
shift : h

# Macros
+w|(tr("+w")>100)|(last("-space")>500), !s, !ctrl ::  +w|15|5, -s|(cs("+w")*1.1), +s|0|0
+s|(tr("+s")>100)|(last("-space")>500), !w, !ctrl ::  +s|15|5, -w|(cs("+s")*1.1), +w|0|0
+a|(tr("+a")>100)|(last("-space")>500), !d, !ctrl ::  +a|15|5, -d|(cs("+a")*1.1), +d|0|0
+d|(tr("+d")>100)|(last("-space")>500), !a, !ctrl ::  +d|15|5, -a|(cs("+d")*1.1), +a|0|0

**Comment by wasiejen** [comment]
when you use:

\```bash
*put your text here then it will be formatted as a code cell and will keep the \<focus> and \<arg> statements
\```

```bash
<arg> -status_indicator=20
<arg> -aliasdelay=50,50
<arg> -tapdelay=10,5
<arg> -nomenu
<arg> -nocontrols

<focus>Counter # Adjust this as needed   <---- !!!!this comment it the problem!!!! it will not be stripped from this line
# Tap Groups

w, s
a, d
# Basic Rebinds

shift : h
# Macros

+w|(tr("+w")>100)|(last("-space")>500), !s, !ctrl :: +w|15|5, -s|(cs("+w")*1.1), +s|0|0
+s|(tr("+s")>100)|(last("-space")>500), !w, !ctrl :: +s|15|5, -w|(cs("+s")*1.1), +w|0|0
+a|(tr("+a")>100)|(last("-space")>500), !d, !ctrl :: +a|15|5, -d|(cs("+a")*1.1), +d|0|0
+d|(tr("+d")>100)|(last("-space")>500), !a, !ctrl :: +d|15|5, -a|(cs("+d")*1.1), +a|0|0

```

**Comment by s0cr4tes** [comment]
Thank you so much - now i made i work :)

Great stuff, keep it up - have a nice day

**Comment by wasiejen** [comment]
Renamed the issue and will be fixed with next update, so comments will be stripped also in \<focus> lines

**Comment by wasiejen** [comment]
fixed in #62
